commands_to_load/Administration.py

```diff
@@ -11,18 +11,29 @@ class Administration():
 	def __init__(self, bot):
 		self.bot = bot
 
+	def realCog(self, cog):
+		for name in cogs:
+			if name["name"] == cog:
+				return [True, name["folder"]]
+		return [False]
+
 	@commands.command()
 	async def load(self, ctx, name):
 		logger.info("[Administration load()] load command detected from "+str(ctx.message.author))
 		if ctx.message.author in discord.utils.get(ctx.guild.roles, name="Bot_manager").members:
+			folder = self.realCog(name)
+			if not folder[0]:
+				await ctx.send("```" + name + " isn't a real cog```")
+				logger.error("[Administration load()] " + str(ctx.message.author) + " tried loading " + name + " which doesn't exist.")
+				return
 			try:
 				logger.info("[Administration load()] "+str(ctx.message.author)+" successfully authenticated")

```

Comment:
this line should be moved to immediately above `folder = self.realCog(name)` because its current place in code implies that the user is successfully authenticated after realCog is called. In other words, place it immediately after the user is authenticated and not some close point after that line of relevant code to make it more relevant.

Reply:
same thing applies for the below functions